src/main/java/seedu/dietbook/calculator/Calculator.java

```diff
@@ -319,6 +319,11 @@ public int calculateRecomendation(Person person) {
         } else {
             recomendation = (int) requirement + 100;
         }
+        if (recomendation < 1000) {
```

Comment:
Perhaps consider refactoring to avoid long methods.

Reply:
OK, I see.